### PR TITLE
fix for pandas 1.2.0

### PR DESCRIPTION
**Pull request by dilpath**
Small change in pandas, `get_filepath_or_buffer` -> `_get_filepath_or_buffer` (not sure if `_get_filepath_or_buffer` should be used in PEtab since it's now named like a private method).

https://github.com/pandas-dev/pandas/pull/37639

### CODE REVIEW

**Review by yannikschaelte — APPROVED** [review]
I guess fine as a hot fix, but should be revised

File: setup.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def absolute_links(txt):
       url='https://github.com/PEtab-dev/PEtab',
       packages=find_packages(exclude=['doc*', 'test*']),
       install_requires=['numpy>=1.15.1',
-                        'pandas>=1.0.1',
+                        'pandas>=1.2.0',
```
Comment (yannikschaelte) [inline setup.py:56]: does get_handle not exist or behave differently for pandas < 1.2.0? but fine to enforce <=1.2.0 I think.

**Review by yannikschaelte — APPROVED** [review]
Nice! I suggest a new release then too as pyabc and pypesto are unhappy.

File: setup.py (hunk 1 above)
Comment (dilpath) [reply setup.py:56]: Does exist but may behave differently (the function that PEtab was using was "merged" with `get_handle` in the pandas PR that prompted this PR -- linked above). I haven't tested this PR with previous pandas versions.

Comment (yannikschaelte) [reply setup.py:56]: thanks for the info, then enforcing :panda_face: >= 1.2.0 simply everywhere.

**Pull request merged**